Buffs/LuluR/LuluR.cs

```diff
@@ -1,44 +1,45 @@
-﻿using System;
-using LeagueSandbox.GameServer.Logic.GameObjects;
-using LeagueSandbox.GameServer.Logic.Scripting;
+﻿using LeagueSandbox.GameServer.Logic.GameObjects;
 using LeagueSandbox.GameServer.Logic.API;
+using LeagueSandbox.GameServer.Logic.Scripting.CSharp;
 
 namespace LuluR
 {
-    class LuluR : BuffGameScript
+    class LuluR : IGameScript
     {
+        GameScriptInformation info;
+        Spell spell;
+        Unit owner;
+        Unit target;
         ChampionStatModifier statMod;
         float healthBefore;
         float meantimeDamage;
         float healthNow;
         float healthBonus;
-
-        public void OnActivate(Unit unit, Spell ownerSpell)
+        public void OnActivate(GameScriptInformation scriptInfo)
         {
+            info = scriptInfo;
+            spell = info.OwnerSpell;
```

Comment:
Adding to this: it cannot be done with an interface, but if an interface isn't necessary we might want to just roll with abstract. Optionally we could add an abstract class that extends from the interface if a scenario like that would make more sense.

For example:
`IGameScript`
`AbstractGameScript : IGameScript`
`LuluR : AbstractGameScript`

Whatever is the most simple and flexible solution should be used, explore some options :)